Software/dist_windows/script32.iss

```diff
@@ -57,18 +57,13 @@ Source: "content/Qt5Network.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "content/Qt5SerialPort.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "content/Qt5Widgets.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "gpl-3.0.txt"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icudt54.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icuin54.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icuuc54.dll"; DestDir: "{app}"; Flags: ignoreversion
```

Comment:
To add on that, it would seem that Qt decided to change their mind since 5.4 regarding ICU: http://wiki.qt.io/Qt-5-QLocale#Platform_Support.

>WIndows 7 and later provides advanced localization functions clearly based on CLDR and broadly comparable to ICU. Windows Vista lacks many of these new API relying mainly on Win32 API's, but with a < 2% market share can be safely ignored provided sensible fallbacks occur. Windows XP only has Win32 API which lack most of the required advanced functions, but basic functionality can still be supported using some fallbacks. The issue comes with what platform Qt is compiled for, runtime checks may not be sufficient to work around missing API calls.

So I guess they use the Windows API natively now, instead of using the ICU libraries?